onnxruntime/core/providers/vitisai/imp/tensor_proto.cc

```diff
@@ -3,10 +3,39 @@
 #include "./tensor_proto.h"
 #include "./vai_assert.h"
 
-#include <cstdint>
-#include <limits>
+#include <iostream>
+#include <string>
+#include <fstream>
+
 
 namespace vaip {
+using Microsoft::WRL::ComPtr;
```

Comment:
I prefer something like this (this is an example, I would like to illustrate the idea):
```c++
using wrl = Microsoft::WRL;
...
wrl::ComPtr<ID3D12Fence> ...;
```
The namespaces have their purpose. Please do not merge them with the global namespace.

Reply:
I found examples inside onnxruntime DirectML execution provider code with using Microsoft::WRL::ComPtr; or they are declaring ComPtr in every call with Microsoft::WRL::ComPtr....;

Example for first case is [here](https://github.com/apwojcik/onnxruntime/blob/92b6e10d37c50638d59620c5a315c6e75b47131c/onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.h#L21)

Example for second case is [here](
https://github.com/apwojcik/onnxruntime/blob/uif2/develop/onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlGraphFusionHelper.cpp).

I changed my code to present ComPtr with Microsoft::WRL::ComPtr.